packages/dd-trace/src/plugins/tracing.js

```diff
@@ -70,10 +70,11 @@ class TracingPlugin extends Plugin {
       childOf = store.span
     }
 
+    meta[COMPONENT] = meta[COMPONENT] || this.component
```

Comment:
Why this change? This is back to altering an object after its creation.

Reply:
What do you mean by altering after creation? I had to add this line to account for the fact that the meta object may already contain the `component` attribute, which in that case, we want to keep the value currently set. Otherwise we need to set the component field to `this.component`.

Reply:
I meant that it could be set on the object passed to `startSpan` instead of separately, but I didn't notice it was set on the meta. Since you'll always want to be setting this regardless of if a `meta` object was passed or not, there should be a default value for `meta` instead of the condition that was added.

Reply:
Reverted this back. Instead, I just set the component field when calling the openTracer startSpan() method. That way, if meta is not set at all, the component field still gets set correctly. If meta is set AND has a component field, meta.component overrides the explicit setting of the 'component' field within the startSpan method.